src/Gensys.jl

```diff
@@ -1,7 +1,10 @@
 module Gensys
+
 include("gensysdt.jl")
 include("gensysct.jl")
-export gensysdt,
-gensysct
+include("types.jl")
```

Comment:
Let's move `types.jl` above the other two because they actually depend on it. It didn't matter in this case because the routines in `gensys(c|d)t.jl` don't actually run when the package is loaded, but it still makes sense!